### PR TITLE
feat: integrate S3 when executing "getLatestValidRouterConfig" from the CLI

### PR DESCRIPTION
**Pull request by JivusAyrus**
- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

File: cli/src/core/config.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ export const config = {
   webURL: process.env.COSMO_WEB_URL || 'https://cosmo.wundergraph.com',
   kcClientId: process.env.KC_CLIENT_ID || 'cosmo-cli',
   kcRealm: process.env.KC_REALM || 'cosmo',
+  cdnURL: process.env.CDN_URL || 'http://127.0.0.1:11000',
```
Comment (StarpTech) [inline cli/src/core/config.ts:35]: The default should always point to cloud production.

File: controlplane/src/core/bufservices/PlatformService.ts [hunk 2]
```diff
@@ -5886,6 +5857,77 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
...
+          return {
+            response: {
+              code: EnumStatusCode.ERR_NOT_FOUND,
+              details: `Federated graph '${req.fedGraphName}' not found`,
+            },
+            token: '',
+          };
+        }
+
+        const expTime = Math.floor(Date.now() / 1000) + 5 * 60;
```
Comment (StarpTech) [inline controlplane/src/core/bufservices/PlatformService.ts:5896]: Could you double check if we validate the exp of jwts in the cdn?

File: proto/wg/cosmo/platform/v1/platform.proto [hunk 3]
```diff
@@ -238,7 +238,6 @@ message GetFederatedGraphByNameResponse {
   Response response = 1;
   FederatedGraph graph = 2;
   repeated Subgraph subgraphs = 3;
```
Comment (StarpTech) [inline proto/wg/cosmo/platform/v1/platform.proto:240]: Correct the other numbers.

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: proto/wg/cosmo/platform/v1/platform.proto (hunk 3 above)
Comment (JivusAyrus) [reply proto/wg/cosmo/platform/v1/platform.proto:240]: Done

File: cli/src/core/config.ts (hunk 1 above)
Comment (JivusAyrus) [reply cli/src/core/config.ts]: Done

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**